src/video_core/texture_cache/texture_cache.h

```diff
@@ -776,6 +787,44 @@ class TextureCache {
         }
     }
 
+    /**
+     * `GetNullSurface` gets a null surface based on a target texture.
+     * @param target, The target of the null surface.
```

Comment:
```suggestion
     * @param target The target of the null surface.
```

Otherwise this is an invalid Doxygen reference